ci/gpu/build.sh

```diff
@@ -55,6 +55,7 @@ conda install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvidia \
       "rapids-notebook-env=$MINOR_VERSION.*" \
       "rapids-doc-env=$MINOR_VERSION.*"
 
+
```

Comment:
can you remove this extraneous line break?
```suggestion
```

Reply:
Thank you @ajschmidt8, I'll do that.